lib/config/presets/http/index.ts

```diff
@@ -0,0 +1,39 @@
+import { logger } from '../../../logger';
+import { ExternalHostError } from '../../../types/errors/external-host-error';
+import { Http } from '../../../util/http';
+import { parseUrl } from '../../../util/url';
+import type { Preset, PresetConfig } from '../types';
+import { PRESET_DEP_NOT_FOUND, extractSubPreset, parsePreset } from '../util';
+
+const id = 'http';
```

Comment:
```suggestion
const id = 'preset';
```

@rarkins WDYT about this id?

Reply:
What's the impact is this change?

Reply:
it's the host type for host rules

Reply:
Seems ok